cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -675,6 +679,95 @@ def test_can_get_correct_usage_info_for_content_groups(self):
 
         self.assertEqual(actual, expected)
 
+    @ddt.data(ModuleStoreEnum.Type.split)
+    def test_can_get_correct_usage_info_with_orphan(self, module_store_type):
+        """
+        Test if content group json updated successfully with usage information even if there is
+        an orphan in content group.
+        """
+        with self.store.default_store(module_store_type):
+
+            split_course = CourseFactory.create()
+
+            # Create user partition
+            split_course.user_partitions = [
+                UserPartition(
+                    0, 'Name 0', 'Description 0',
+                    [Group(0, 'Group A'), Group(1, 'Group B'), Group(2, 'Group C')],
+                    scheme=None, scheme_id='cohort'
+                )]
+            split_course.save()
+
+            # Create a problem with content group
+            chapter = self.store.create_child(
```

Comment:
You can use [`ItemFactory`](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/tests/factories.py#L277-L278), which will probably make your life a lot easier